ecs-cli/modules/config/params_test.go

```diff
@@ -122,8 +134,8 @@ func TestNewCliParamsWhenPrefixesPresent(t *testing.T) {
 	rdwr := &mockReadWriter{isKeyPresentValue: true}
 	params, err := NewCLIParams(context, rdwr)
 	assert.NoError(t, err, "Unexpected error when getting new cli params")
-	assert.Empty(t, params.ComposeServiceNamePrefix, "Expected ComposeServiceNamePrefix to be empty")
-	assert.Equal(t, clusterName, params.CFNStackName, "Expected CFNStackName to be default")
+	assert.Equal(t, composeServiceNamePrefix, params.ComposeServiceNamePrefix, "Expected ComposeServiceNamePrefix to be empty")
+	assert.Equal(t, cfnStackName, params.CFNStackName, "Expected CFNStackName to be default")
```

Comment:
Should the description be "Expected CFNStackName to match" since you specified a name, it wouldn't be default?